### PR TITLE
[PF-163] Increase service usage http request timeout

### PR DESCRIPTION
**Pull request by yonghaoy**

### CODE REVIEW

File: src/main/java/bio/terra/rbs/app/configuration/CrlConfiguration.java [hunk 1]
```diff
@@ -114,6 +116,15 @@ public CloudBillingClientCow cloudBillingClientCow() throws IOException {
   @Bean
   @Lazy
   public ServiceUsageCow serviceUsageCow() throws GeneralSecurityException, IOException {
+    new ServiceUsageCow(clientConfig(),
+            new ServiceUsage.Builder(
+                    GoogleNetHttpTransport.newTrustedTransport(),
+                    Defaults.jsonFactory(),
+                    setHttpTimeout(
```
Comment (wchamber) [inline src/main/java/bio/terra/rbs/app/configuration/CrlConfiguration.java:123]: the comment on setHttpTimeout references `ResourceManager`, but this isn't ResourceManager. make the comment more generic.

**Review by wchamber — COMMENTED** [review]
How'd you figure out this was the issue?

File: src/main/java/bio/terra/rbs/app/configuration/CrlConfiguration.java (hunk 1 above)
Comment (yonghaoy) [reply src/main/java/bio/terra/rbs/app/configuration/CrlConfiguration.java:123]: done

**Comment by yonghaoy** [comment]
@wchamber Don't know how to reply in previous thread. 
How I figure it out: There is one previous test failiure caused by this timeout. And Greg also mentioned service usage calls might be the most time consuming Gcp calls.

**Pull request closed without merging**